code/go/0chain.net/blobbercore/allocation/allocationchange.go

```diff
@@ -7,7 +7,9 @@ import (
 	"0chain.net/blobbercore/datastore"
 	"0chain.net/blobbercore/reference"
 	"0chain.net/core/common"
+	"0chain.net/core/logging"
```

Comment:
not a fan of [dot](https://github.com/golang/go/wiki/CodeReviewComments#import-dot) [import](https://github.com/golang/go/issues/29326), but most of the codebase uses it, so I can change if necessary.